conf/solr/7.2.1/solrconfig.xml

```diff
@@ -0,0 +1,1389 @@
+<?xml version="1.0" encoding="UTF-8" ?>

```

Comment:
I'm confused about why all these Solr 7.2.1 files are being added back in. Did you branch before we upgraded to Solr 7.3.0? If so, please merge the latest from "develop" into this branch and hopefully these additions will go away from the "diff".